hyperspy/_signals/lazy.py

```diff
@@ -21,7 +21,7 @@
 
 import numpy as np
 import dask.array as da
-import dask.delayed as dd
+#import dask.delayed as dd
```

Comment:
Oops, sorry! Must have forgotten to exclude that from one of my more recent commits.